src/Menu.php

```diff
@@ -38,7 +38,7 @@ public function addItem($item = null, $action = null)
         $item = $this->add($item)->setElement('a');
 
         if (is_array($action)) {
-            $action = $this->app->url($action);
+            $action = $this->url($action);
```

Comment:
need to test this change

Reply:
Tested with 

``` php
    $frame->add(['Menu'])->addItem('Test', ['foo'=>'bar']);
```

but no good demo here.